src/renderer/components/image-button/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC, MouseEventHandler } from 'react';
+import Image from 'react-bootstrap/Image';
+
+interface ImageButtonProps {
+  onClick?: MouseEventHandler<HTMLElement>;
+  img: string;
```

Comment:
To get visibility for previous comments. I'd recommend calling `img` just `src` and `passedWidth` just `width`. Also makes things more self-explainatory, especially you didn't add any documentation here. Using the same name as `Image` helps.

Reply:
That said, still a good idea to add documentation since this will be a public interface.

Reply:
I got the img variable to switch to src - but still having a problem with width - see comment above for a screenshot of the error

where should I add the documentation for the image button in the repo?

Reply:
Above.

```
/**
  * JsDoc here.
  */
export interface ImageButtonProps {
  // ...
}

/**
  * JsDoc here.
  */
export const ImageButton: FC<ImageButtonProps> = //...
```